README.md

```diff
@@ -23,7 +23,7 @@ directly, via a `Timer`, or indirectly via a `Buffer`.
 ## Primary Use
 
 ```js
-import Igniter from 'igniter-core';
```

Comment:
I think the npm package and the import from npm could be `igniter`, however creating a build that supports multiple internal packages also seems nice for the future.

Totally happy to export an API to npm as `igniter`. I think that should perhaps be a package or a default export that re-exports parts of `runtime`.